tests/integration/views/surveys/help/test_surveys_help_info_about_this_survey.py

```diff
@@ -0,0 +1,384 @@
+import unittest
+from unittest.mock import patch
+
+import requests_mock
+
+from frontstage import app
+from tests.integration.mocked_services import encoded_jwt_token, survey_eq, survey, survey_list_todo, url_banner_api, \
+    survey_rsi
+
+
+class TestSurveyHelpInfoAboutThisSurvey(unittest.TestCase):
+    def setUp(self):
+        self.app = app.test_client()
+        self.app.set_cookie('localhost', 'authorization', 'session_key')
+        self.headers = {
+            "Authorization": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoicmluZ3JhbUBub3d3aGVyZS5jb20iLCJ1c2VyX3Njb3BlcyI6WyJjaS5yZWFkIiwiY2kud3JpdGUiXX0.se0BJtNksVtk14aqjp7SvnXzRbEKoqXb8Q5U9VVdy54"
+            # NOQA
+        }
+        self.patcher = patch('redis.StrictRedis.get', return_value=encoded_jwt_token)
+        self.patcher.start()
+
+    def tearDown(self):
+        self.patcher.stop()
+
+    @requests_mock.mock()
+    @patch('frontstage.controllers.survey_controller.get_survey_by_short_name')
```

Comment:
Can you change this (and all the others in this) to use a mock url instead of patching the function?
`mock_request.get(url_get_survey_by_short_name_eq, status_code=200, json=survey_eq)`

I think if you import `url_get_survey_by_short_name_eq` then this should just work, and you can remove the patch.
If it's more effort then that then don't worry about it.  It'd be nice not have 2 ways to do similar things, but it's also not a big deal.

Reply:
cool - I will leave it for now :-)